sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -59,7 +59,7 @@ def __init__(
         credential: Optional[TokenCredential] = None,
         *,
         api_version: Optional[str] = None,
-        audience: str = AZURE_RESOURCE_MANAGER_PUBLIC_CLOUD,
+        audience: str = DEFAULT_AUDIENCE,
```

Comment:
just checking - will the default value changing impact folks who were used to having the old value ?

Reply:
No. The old value only works in public cloud, the new one works for all clouds.

Reply:
ah so it will be a seamless change :)